test_conformance/multiple_device_context/testBase.h

```diff
@@ -16,16 +16,8 @@
 #ifndef _testBase_h
 #define _testBase_h
 
-#include "harness/compat.h"
-
-#include <stdio.h>
-#include <string.h>
-#include <sys/types.h>
-#include <sys/stat.h>
-
-#include "procs.h"
+#include "harness/conversions.h"
```

Comment:
Not going to lose sleep over this, but it does seem a little odd to include the conversions header here.

Reply:
Harness headers are a mess generally. They will need fixing but this is better tackled with a dedicated PR IMO.